Userland/Applications/HexEditor/HexEditor.cpp

```diff
@@ -724,15 +732,16 @@ void HexEditor::paint_event(GUI::PaintEvent& event)
             };
 
             painter.fill_rect(text_background_rect, background_color_text);
-            auto character = String::formatted("{:c}", isprint(cell_value) ? cell_value : '.').release_value_but_fixme_should_propagate_errors();
 
             if (m_edit_mode == EditMode::Text)
                 draw_cursor_rect();
 
+            auto character = is_ascii_printable(cell.value) ? cell.value : '.';
+            auto character_sv = StringView { reinterpret_cast<char const*>(&character), 1 };
```

Comment:
Can't you just make `character` be a `char const` instead of casting?

Reply:
Yup! That's better. :^)